### PR TITLE
Identify model from description

### LINKED ISSUE #466: Create relationship between Diagram description and model
As suggested in #465 the model should be determined from a Diagram description. 

After this feature is implemented the diagram visualization wizard has to be modified: The two browser fields (project, model) should be deleted, only the tree-view has to be present.

### PR DESCRIPTION
**Pull request by kmozsi**
Resolves #466.

- The model is determined from layout description/deployment configuration.
- The model selection fields are deleted from the visualization and cpp gen wizard, since the model is identified automatically.

### CODE REVIEW

File: dev/plugins/hu.elte.txtuml.export.papyrus/src/hu/elte/txtuml/export/papyrus/wizardz/TxtUMLVisuzalizeWizard.java [hunk 1]
```diff
@@ -67,106 +76,126 @@ public void addPages() {
...
-					TxtUMLLayoutDescriptor layoutDescriptor = null;
-					try {
-						layoutDescriptor = exporter.exportTxtUMLLayout();
+		List<IType> txtUMLLayout = selectTxtUmlPage.getTxtUmlLayout();
+		Map<Pair<String, String>, List<IType>> layoutConfigs = new HashMap<>();
+		for (IType layout : txtUMLLayout) {
+			IType innerLayoutClass = null;
+			try {
+				innerLayoutClass = Stream.of(layout.getTypes()).findFirst().get();
+			} catch (JavaModelException e) {
```
Comment (kmozsi) [inline dev/plugins/hu.elte.txtuml.export.papyrus/src/hu/elte/txtuml/export/papyrus/wizardz/TxtUMLVisuzalizeWizard.java]: This exception is thrown when the enclosing type (layout) is not present - removed after the layout selection and before the diagram generation. I don't believe it is possible, leastwise i cannot do that.

**Comment by dobreffandras** [comment]
@kmozsi  please review my changes.
@nagyandras95  do you approve?

**Comment by kmozsi** [comment]
@dobreffandras, looks good, thank you. I've also tried the new functionality, it works. 👍 
@nagyandras95, please review the c++ gen wizard related commits.

**Comment by kmozsi** [comment]
This problem occurs when 2 different projects contain the same model package name (here: machine1.j.model). Somehow we should tell which project is referenced in the layout description/configuration. 

Currently, when the fully qualified names of the referenced model classes are the same in both projects, we simply choose the first one, which is definitely not a correct solution.

**Comment by kmozsi** [comment]
@nagyandras95, please review the changes.

File: examples/demo/machine/src/machine1/j/DiagDescWithoutModel.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package machine1.j;
+
+import hu.elte.txtuml.api.layout.ClassDiagram;
+import hu.elte.txtuml.api.layout.Right;
+
+public class DiagDescWithoutModel extends ClassDiagram {
```
Comment (djnemeth) [inline examples/demo/machine/src/machine1/j/DiagDescWithoutModel.java]: Was this file intentionally pushed?

Comment (nagyandras95) [inline examples/demo/machine/src/machine1/j/DiagDescWithoutModel.java]: I have transposed to the test machine model.

**Pull request merged**